package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "instaqchallenge",
+  "version": "1.0.0",
+  "description": "Um projeto backend focado na execução do desafio proposto pela instaQ",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "ts-node src/server.ts"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/node": "^22.7.4",
```

Geralmente setamos uma versão especifica, para que o projeto não quebre devido alguma atualização de lib externa.
```suggestion 
    "@types/node": "22.7.4",
```

Após setar a versão especifica é preciso reinstalar as libs.